Makefile.am

```diff
@@ -24,6 +24,11 @@ tests/test-switch-server: tests/test_switch_server.c \
 tests/test-fd-leaks: tests/test_fd_leaks.c \
   tools/*.h tools/*.c
 
+clean-local:
```

Comment:
The coverage files should only exist if we are building with coverage right? Thus I think this might need to be inside the section below?
```automake
if GCOV_ENABLED

endif
```

You should also put a `-` in front of each of the rm lines, so if no files exist (and thus rm fails) it still continues. See http://www2.ipp.mpg.de/~dpc/gmake/make_47.html